webroot/js/app/components/relation-view/relation-view.js

```diff
@@ -101,12 +101,9 @@ Vue.component('relation-view', {
          *
          * @returns {void}
          */
-        removeRelations(id, type) {
-            if (!this.containsId(this.removedRelated, id)) {
-                this.removedRelated.push({
-                    id,
-                    type,
-                });
+        removeRelations(related) {
+            if (!this.containsId(this.removedRelated, related.id)) {
```

Comment:
Better verifying `related` and `related.id`, before passing `related.id` to `containsId`.
I.e.:
```
if (!related || !related.id) {
    return;
}
if (!this.containsId(this.removedRelated, related.id)) {
```

Reply:
giustissimo!